dot/core/interface.go

```diff
@@ -29,7 +29,7 @@ type RuntimeInstance interface {
 	Exec(function string, data []byte) ([]byte, error)
 	SetContextStorage(s runtime.Storage)
 	GetCodeHash() common.Hash
-	Version() (runtime.Version, error)
+	Version() (version runtime.Version)
```

Comment:
was not these changes already merged in another branch? Should you rebase this branch with development?

Reply:
Indeed, this is fixed up, sorry about that!